app/routes/identifikation/index.tsx

```diff
@@ -53,15 +52,14 @@ export const loader: LoaderFunction = async ({ request }) => {
 
   const bundesIdentIsOnline =
     !flags.isBundesIdentDisabled() && !flags.isBundesIdentDown();
-  const session = await getSession(request.headers.get("Cookie"));
-  const hasPrimaryOptionShown = Boolean(session.get("hasPrimaryOptionShown"));
-  const shouldShowBundesidentPrimayOptionPage = showBundesidentPrimayOptionPage(
+
+  const shouldShowPrimaryOption = hasPrimaryOptionEligibility(
+    request.url,
     bundesIdentIsOnline,
-    hasPrimaryOptionShown,
     dbUser
   );
 
-  if (shouldShowBundesidentPrimayOptionPage) {
+  if (shouldShowPrimaryOption) {
```

Comment:
If someone goes back to that page without any query parameters or a link (e.g. in the documentation) is still set to identification, they will always be shown the primary option, right? 
Which is fine...

Reply:
yes correct, they will see the primary option which I think is what we want